src/CoreBundle/Resources/config/data_access.xml

```diff
@@ -91,5 +91,26 @@
             <argument type="service" id="database_connection"/>
         </service>
 
+        <service class="ChameleonSystem\CoreBundle\DataAccess\DataAccessCmsMasterPagedefFile" id="chameleon_system_core.data_access_cms_master_pagedef_file" public="false">
+            <argument id="chameleon_system_core.util.input_filter" type="service"/>
+            <argument id="chameleon_system_core.global" type="service"/>
+        </service>
+        <service class="ChameleonSystem\CoreBundle\DataAccess\DataAccessCmsMasterPagedefCacheDecorator" id="chameleon_system_core.data_access_cms_master_pagedef_file_cache_decorator" decorates="chameleon_system_core.data_access_cms_master_pagedef_file" public="false">
```

Comment:
It would be best to have all these parameters in a new line each.

(for example the "decorates" is rather important here - but not visible without scrolling)

Reply:
formatted